ocs_ci/ocs/acm/acm.py

```diff
@@ -513,6 +514,20 @@ def import_clusters_via_cli(clusters):
             column="JOINED",
             resource_name=cluster[0],
         )
+
+        log.info("Create the Klusterlet addon configuration")
+        klusterlet_config = templating.load_yaml(constants.ACM_HUB_KLUSTERLET_YAML)
+        klusterlet_config["metadata"]["name"] = cluster[0]
+        klusterlet_config["metadata"]["namespace"] = cluster[0]
+        klusterlet_config_obj = OCS(**klusterlet_config)
+        klusterlet_config_obj.apply(**klusterlet_config)
+
+        log.info("Wait for up to 60s for addon pods to be initialized")
+        time.sleep(60)
```

Comment:
@parikshithb , Instead of sleep can we wait on the resource with a state ?

Reply:
Creating klusterlet yaml, creates addon pods in "open-cluster-management-agent-addon", waiting here for these pods to be in initializing/ContainerCreating before validating all of them to be in running state by calling wait_for_pods_to_be_running.

Reply:
@parikshithb why is sleep needed in that case? Should we add that time to the timeout we pass to wait_for_pods_to_be_running()?

Reply:
> @parikshithb why is sleep needed in that case? Should we add that time to the timeout we pass to wait_for_pods_to_be_running()?

I agree with @ebenahar , we should pass the timeout while waiting for pods.

Reply:
Agree, made the change and validated.